### ISSUE TITLE
"brew install fish" fails on ubuntu

### ISSUE THREAD

**Issue by tdsmith** [Linuxbrew/linuxbrew#485]
_From @jab on July 16, 2015 0:38_

looks like the formula is looking for `/usr/bin/sed` (which is where it lives on OS X) but on Ubuntu it's at `/bin/sed`.

```
/usr/bin/sed <fish.pc.in >fish.pc \
		-e "s,@sysconfdir\@,/home/wam/.linuxbrew/Cellar/fish/2.2.0/etc,g" \
		-e "s,@datadir\@,/home/wam/.linuxbrew/Cellar/fish/2.2.0/share,g" \
		-e "s,@docdir\@,/home/wam/.linuxbrew/Cellar/fish/2.2.0/share/doc/fish,g" \
		-e "s|@configure_input\@|fish.pc, generated from fish.pc.in by the Makefile. DO NOT MANUALLY EDIT THIS FILE!|g" \
		-e "s,@prefix\@,/home/wam/.linuxbrew/Cellar/fish/2.2.0,g" \
		-e "s,@fish_build_version\@,2.2.0,g" \

/bin/sh: 1: /usr/bin/sed: not found
make: *** [fish.pc] Error 127
```

_Copied from original issue: Homebrew/homebrew#41760_

**Comment by puppe** [comment]
Duplicate of #482

**Comment by rwhogg** [comment]
This issue can probably be closed now that #482 is closed.

**Comment by sjackman** [comment]
Closing. Thanks, Bob.
